Makefile

```diff
@@ -1,6 +1,10 @@
 manage = python manage.py
 install = pip install
 
+docker:
+	sed -e 's/-r common.txt//' requirements/dev.txt \
+	  | cat requirements/common.txt - > requirements.txt
```

Comment:
So this just creates a `requirements.txt` when you run `make docker`?

I think it would make more sense to just commit a `requirements.txt` that looked liked this

    # Dev requirements
    # where docker expects them to be
    -r requirements/common.txt
    -r requirements/dev.txt